### PR TITLE
DP Panel: Drop protocol check when creating a new item

### PR DESCRIPTION
**Pull request by norascheuch**
## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/test/unit-tests/common/github-url-identifier-helper.test.ts [hunk 1]
```diff
@@ -35,6 +34,7 @@ describe("github url identifier helper", () => {
 
     it("should handle valid urls", () => {
       expect(getNwoFromGitHubUrl("https://github.com/foo/bar")).toBe("foo/bar");
+      expect(getNwoFromGitHubUrl("http://github.com/foo/bar")).toBe("foo/bar");
```
Comment (shati-patel) [inline extensions/ql-vscode/test/unit-tests/common/github-url-identifier-helper.test.ts:37]: Could we also add a check for just `"github.com/foo/bar"`?

**Review by shati-patel — APPROVED** [review]
Looks good! 🎉 Would you mind adding a short PR description?

File: extensions/ql-vscode/test/unit-tests/common/github-url-identifier-helper.test.ts (hunk 1 above)
Comment (norascheuch) [reply extensions/ql-vscode/test/unit-tests/common/github-url-identifier-helper.test.ts:37]: good idea!

**Comment by norascheuch** [comment]
I think the whole requirement is BS but it should work now.

**Pull request merged**